.github/pull_request.yml

```diff
@@ -0,0 +1,34 @@
+name: Pull Request
+on: [pull_request, push]
+
+jobs:
+  make_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+
+      # See https://debian.neo4j.com/
+      - name: Neo4j setup
+        run: |
+          wget -O - https://debian.neo4j.com/neotechnology.gpg.key | sudo apt-key add -
+          echo 'deb https://debian.neo4j.com stable 3.5' | sudo tee /etc/apt/sources.list.d/neo4j.list
+          sudo apt-get update
+
+      - name: Install Neo4j
+        run: sudo apt-get install neo4j
+
+      - name: Disable auth for neo4j
+        run: |
+          sudo sed -i 's/#dbms.security.auth_enabled=false/dbms.security.auth_enabled=false/g' /etc/neo4j/neo4j.conf
+          sudo service restart neo4j
+
+      - name: Install cartography
```

Comment:
You can wrap this in a caching action for improved installation speeds https://github.com/actions/cache/blob/main/examples.md#python---pip